config/certification-settings.ts

```diff
@@ -54,9 +54,9 @@ export const certIds = {
 };
 
 export const completionHours = {
-  [certTypes.frontEnd]: 400,
-  [certTypes.backEnd]: 400,
-  [certTypes.dataVis]: 400,
+  [certTypes.frontEnd]: 300,
+  [certTypes.backEnd]: 300,
+  [certTypes.dataVis]: 300,
```

Comment:
So the things with these is that, long time ago we had only 3 certs. And presumably would take users longer to complete each certification.

Reply:
Although we are sunsetting that part of freeCodeCamp, there might be people that find issue with this change.

I think we move forward with changing everything to 300 and revert if an issue is raised.